android/app/build.gradle

```diff
@@ -135,10 +128,13 @@ task copyDownloadableDepsToLibs(type: Copy) {
 }
 
 apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+apply from: file("../mendixnative-release/mendix.gradle"); applyMendixGradle(project)
```

Comment:
Could we do our stuff before react's?

Reply:
I think so we don't really relate to what RN does.

Reply:
At least on the iOS side they check if a dependency has already been added so they do not add to it again, and they expressly advised running complimentary scripts before theirs...

Reply:
doesn't matter for gradle